packages/workspace/src/node/default-workspace-server.spec.ts

```diff
@@ -0,0 +1,109 @@
+/********************************************************************************
+ * Copyright (C) 2022.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { Container, ContainerModule } from '@theia/core/shared/inversify';
+import { MockEnvVariablesServerImpl } from '@theia/core/lib/browser/test/mock-env-variables-server';
+import { EnvVariablesServer } from '@theia/core/lib/common/env-variables';
+import { FileUri } from '@theia/core/lib/node';
+import { CommonWorkspaceUtils } from '../common';
+import { DefaultWorkspaceServer, WorkspaceCliContribution } from './default-workspace-server';
+import { expect } from 'chai';
+import * as temp from 'temp';
+import * as fs from 'fs';
+import * as path from 'path';
+
+/* eslint-disable max-len, no-unused-expressions */
```

Comment:
The suppression should be removed, and the tests should be updated accordingly.

Reply:
Kept the no-unused-expressions suppression as that one is required for soe chai calls, e.g. expect(recent).to.be.empty. Or should i add the suppression to every occurrence instead?